test/extended/util/framework.go

```diff
@@ -1709,10 +1710,11 @@ func (r *podExecutor) Exec(script string) (string, error) {
 	var out string
 	waitErr := wait.PollImmediate(1*time.Second, 3*time.Minute, func() (bool, error) {
 		var err error
-		out, err = r.client.Run("exec").Args(r.podName, "--", "/bin/bash", "-c", script).Output()
+		out, err = e2e.RunHostCmd(r.client.Namespace(), r.podName, script)
```

Comment:
Have you researched any possible issues that might occur by replacing `bin/bash` with `bin/sh -x`?

Reply:
In the container images I've checked so far, `bin/sh` is the same as `bin/bash`, that's a Red Hat thing. Even if it wasn't the same, there shouldn't be a problem, unless someone was using specific features of `bin/bash` in some sort of advanced script command, such as arrays, certain wildcards, or `[[` vs. `[` if statements.

But yes, either way we should be fine, since they are the same.